src/js/controller/controller.js

```diff
@@ -336,13 +338,37 @@ Object.assign(Controller.prototype, {
         function _checkPlayOnViewable(model, viewable) {
             if (model.get('playOnViewable')) {
                 if (viewable) {
-                    _autoStart();
-                } else if (OS.mobile || (model.get('autoPause') && model.get('autoPause').viewability)) {
+                    _autoStart('viewable');
+                } else if (OS.mobile) {
                     _this.pause({ reason: 'autostart' });
                 }
             }
         }
 
+        function _checkPauseOnViewable(model, viewable) {
+            const instream = _this._instreamAdapter;
+
+            // If player left view and came back before ad completes,
+            // the player should begin/resume playing content.
+            if (viewable && instream && instream.noResume) {
```

Comment:
You could use `_getAdState() === PAUSED` instead of checking `instream.noResume`.

Reply:
Is `instream.noResume` always `true` when the `adState` is paused?

Reply:
No, but it could be if you want the player to not call `play` to resume after an ad break. I would expect that to change either here or in  @vseventer changes https://github.com/jwplayer/jwplayer/pull/3295 if you need it to be (`instream.noResume` shouldn't stop playback or go to idle though - that should only happen when calling `jwplayer().stop()`

Currently `instream.noResume` defaults to `false` because ads play as a results of content playback and should resume content when complete.